exchanges/kline/types.go

```diff
@@ -9,27 +10,40 @@ import (
 
 // Consts here define basic time intervals
 const (
-	OneMin     = time.Minute
-	ThreeMin   = 3 * time.Minute
-	FiveMin    = 5 * time.Minute
-	FifteenMin = 15 * time.Minute
-	ThirtyMin  = 30 * time.Minute
-	OneHour    = 1 * time.Hour
-	TwoHour    = 2 * time.Hour
-	FourHour   = 4 * time.Hour
-	SixHour    = 6 * time.Hour
-	TwelveHour = 12 * time.Hour
-	OneDay     = 24 * time.Hour
-	ThreeDay   = 72 * time.Hour
-	OneWeek    = 168 * time.Hour
+	OneMin         = Interval(time.Minute)
+	ThreeMin       = 3 * OneMin
+	FiveMin        = 5 * OneMin
+	TenMin         = 10 * OneMin
+	FifteenMin     = 15 * OneMin
+	ThirtyMin      = 30 * OneMin
+	OneHour        = Interval(1 * time.Hour)
+	TwoHour        = 2 * OneHour
+	FourHour       = 4 * OneHour
+	SixHour        = 6 * OneHour
+	EightHour      = 8 * OneHour
+	TwelveHour     = 12 * OneHour
+	TwentyFourHour = 24 * OneHour
+	OneDay         = TwentyFourHour
+	ThreeDay       = 3 * OneDay
+	SevenDay       = 7 * OneDay
+	Fifteenday     = 15 * OneDay
```

Comment:
What uses this scale?

Reply:
[Kraken](https://www.kraken.com/features/api#get-ohlc-data) have 15 days (21600 minutes)

Reply:
Yeah I saw that later on will resolve.